src/firefly/plans/xafs_scan.py

```diff
@@ -304,6 +384,49 @@ def update_regions(self):
         elif diff_region_num > 0:
             self.add_regions(diff_region_num)
 
+    def update_total_time(self):
+        # Summing total_time for all checked regions directly within the sum function using a generator expression
+        kEranges_all = [
+            region_i.kErange
+            for region_i in self.regions
+            if region_i.region_checkbox.isChecked()
+        ]
+
+        # prevent end points are smaller than start points
+        try:
+            _, exposures = merge_ranges(*kEranges_all, sort=True)
+            total_time_per_scan = exposures.sum()
+        except:
```

Comment:
Catch the specific exception you're expecting. No bare except statements.

Reply:
solved:` except ValueError`